tls/s2n_crl.h

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+
+#pragma once
+
+#include "api/s2n.h"
+
+#include <openssl/x509v3.h>
+
+struct s2n_x509_crl {
```

Comment:
I think it isn't useful. I only named it that because of the openssl struct. There aren't other CRL formats that I'm aware of. I renamed this to `s2n_crl`.